pkg/cloudprovider/providers/azure/azure_file.go

```diff
@@ -23,6 +23,10 @@ import (
 	azs "github.com/Azure/azure-sdk-for-go/storage"
 )
 
+const (
```

Comment:
Please put this onto a single line instead of using a block since there is only one constant.